src/__tests__/lib/containers/TOC.test.tsx

```diff
@@ -19,32 +17,31 @@ describe('renders without crashing', () => {
     SynapseClient.getEntityWiki = jest.fn(() =>
       Promise.resolve({ markdown: '${toc}\n#Heading1' }),
     )
-    const tree = await mount(
-      <MarkdownSynapse ownerId={mockOwnerId} wikiId={mockWikiId} />,
-      {
-        wrappingComponent: SynapseTestContext,
-      },
-    )
-    await delay(_TIME_DELAY)
-    expect(tree.find('div.markdown')).toHaveLength(1)
-    // peculiar behavior below where only usng .render() works
-    expect(tree.render().find('a.link.toc-indent1')).toHaveLength(1)
-    expect(tree.render().find('h1#SRC-header-1')).toHaveLength(1)
+
+    render(<MarkdownSynapse ownerId={mockOwnerId} wikiId={mockWikiId} />, {
+      wrapper: createWrapper(),
+    })
+
+    // Render a link in the TOC that points to the corresponding heading element
+    const tocLink = await screen.findByRole<HTMLAnchorElement>('link')
+    const heading = await screen.findByRole<HTMLHeadingElement>('heading')
+    expect(tocLink).toHaveClass('link toc-indent1')
+
+    // TODO: Test that the link points to the header
```

Comment:
Not a new missing test, this was missing before. The widget uses some sophisticated scrolling library, so we can't just check the `href` attribute of the element.